cmd/api/api.go

```diff
@@ -136,3 +148,126 @@ func checkDB(database *database.SQLdb, timeout time.Duration) error {
 
 	return database.DB.PingContext(ctx)
 }
+
+// getFiles returns the files from the database for a specific user
+func getFiles(c *gin.Context) {
+
+	log.Debugf("request files in project")
+	c.Writer.Header().Set("Content-Type", "application/json")
+	// Get user ID to extract all files
+	userID, err := getUserFromToken(c.Writer, c.Request)
+	if err != nil {
+		// something went wrong with user token
+		c.JSON(500, err.Error())
+
+		return
+	}
+
+	files, err := Conf.API.DB.GetUserFiles(userID)
+	if err != nil {
+		// something went wrong with querying or parsing rows
+		c.JSON(500, err.Error())
+
+		return
+	}
+
+	// Return response
+	c.JSON(200, files)
+}
+
+// getUserFromToken parses the token, validates it against the key and returns the key
+func getUserFromToken(w http.ResponseWriter, r *http.Request) (string, error) {
+	// Check that a token is provided
+	tokenStr, err := getToken(r.Header.Get("Authorization"))
+	if err != nil {
+		log.Error("authorisation header missing frm request")
```

Comment:
```suggestion
		log.Error("Authorization header missing from request")
```

Typo fix, use actual name of expected header.